### PR TITLE
Added icon for taking pictures

### PR DESCRIPTION
**Pull request by teamagam5**
Added icon for taking pictures

### CODE REVIEW

File: DailySelfie/app/src/main/res/menu/options_menu.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+<!-- "Take Picture", should appear as action button if possible -->
+<item
+    android:id="@+id/action_take_picture"
+    android:icon="@drawable/ic_add_a_photo_black_24dp"
+    android:title="@string/action_take_picture"
```
Comment (teamagam1) [inline DailySelfie/app/src/main/res/menu/options_menu.xml]: We name strings with their context in the prefix. for example, in this case, the would be:
@string/activity_main_action_take_picture

**Review by teamagam1 — CHANGES_REQUESTED** [review]

File: DailySelfie/app/src/main/res/menu/options_menu.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    app:showAsAction="ifRoom"/>
```
Comment (teamagam1) [inline DailySelfie/app/src/main/res/menu/options_menu.xml]: are you sure you need this attribute?

Comment (teamagam1) [inline DailySelfie/app/src/main/res/menu/options_menu.xml]: are you sure you need this attribute?

Comment (teamagam5) [reply DailySelfie/app/src/main/res/menu/options_menu.xml]: thought maybe the screen's too small so use the overflow

**Pull request merged**